candidates/views/api.py

```diff
@@ -433,3 +433,8 @@ class ComplexPopoloFieldViewSet(viewsets.ModelViewSet):
     queryset = extra_models.ComplexPopoloField.objects.order_by('id')
     serializer_class = serializers.ComplexPopoloFieldSerializer
     pagination_class = ResultsSetPagination
+
+class PersonRedirectViewSet(viewsets.ModelViewSet):
```

Comment:
I think that's fine as in future I can imagine the API accepting non-GET requests. However this endpoint should never accept any form of modification, so that should be encoded beyond a change to `DEFAULT_PERMISSION_CLASSES`